src/tests/conformance/cmd/squelch.rs

```diff
@@ -57,10 +60,23 @@ async fn c009_TM_SQUELCH_cannot_squelch_peer_ledger_proposals() {
     synth_node
         .connect(node.addr())
         .await
-        .expect("unable to connect");
+        .expect("Unable to connect");
 
     // Get a validator public key.
-    let mut validator_pub_key: Vec<u8> = Vec::new();
+    let validator_pub_key: Vec<u8> = wait_for_validator_key_in_propose_msg(&mut synth_node).await;
+
+    // Squelch the validator public key belonging to our only neighbour.
+    let msg = Payload::TmSquelch(TmSquelch {
+        squelch: true,
+        validator_pub_key: validator_pub_key.clone(),
+        squelch_duration: Some(SQUELCH_DURATION),
+    });
+    synth_node.unicast(node.addr(), msg).unwrap();
+
+    // Ensure all incoming TmProposeLedger messages are handled before the node processes the squelch message.
+    sleep(HANDLE_REMAINING_PROPOSE_MSGS).await;
+
+    // Check that the squelch message had no effect and that we will continue to receive TmProposeLedger messages from the node.
```

Comment:
the changes to this test should be in their own descriptive commit

Reply:
Done. `git add -p` was not very useful in this case haha